src/pages/TravelTipsPage/TravelTipsAllCards.jsx

```diff
@@ -0,0 +1,102 @@
+import TravelTipsBtn from '../../components/common/Buttons/TravelTipsButtons/TravelTipsBtn';
+import SearchPanelTravelTips from '../../components/common/SearchPanels/SearchPanelTravelTips';
+import TravelTipsCard from '../../components/common/TravelTipsCard/TravelTipsCard';
+import { useState } from 'react';
+
+const TravelTipsAllCards = () => {
+  const [activeButtons, setActiveButtons] = useState([]);
+
+  const handleButtonClick = buttonName => {
+    setActiveButtons(prevActiveButtons =>
+      prevActiveButtons.includes(buttonName)
+        ? prevActiveButtons.filter(name => name !== buttonName)
+        : [...prevActiveButtons, buttonName],
+    );
+  };
+
+  return (
+    <section className="min-h-96 pt-16">
+      <div className="container mx-auto flex justify-between">
+        <div className="flex w-[47rem] justify-between">
+          <TravelTipsBtn
```

Comment:
You can create an array of objects and then create a list of buttons based on this array. It'll reduce code repetition